dev/io.openliberty.microprofile.openapi.3.0.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.openliberty.microprofile.openapi30.internal</groupId>
+    <artifactId>tck.runner.tck</artifactId>
+    <version>3.0</version>
+    <packaging>jar</packaging>
+
+    <name>MicroProfile OpenAPI TCK Runner</name>
+
+    <properties>
+        <microprofile.openapi.version>3.0-RC4</microprofile.openapi.version>
+
+        <surefire.version>2.17</surefire.version> <!-- Any changes to the surefire version must be tested against ZOS-->
+
+        <arquillian.version>1.7.0.Alpha9</arquillian.version>
+        <arquillian-wlp.version>2.0.0</arquillian-wlp.version>
+
+        <suiteXmlFile>${defaultSuiteFiles}</suiteXmlFile>
+        <targetDirectory>${project.basedir}/target</targetDirectory>
+
+        <testng.version>7.4.0</testng.version>
+
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- Needed for ZOS. Defines the version of this artifact that arquillian-liberty-managed should use -->
+            <!-- This doesn't exist for JEE 9 so leaving it out for now, will have to see what happens on ZOS -->
+            <!-- <dependency>
+                <groupId>org.jboss.spec.javax.servlet</groupId>
+                <artifactId>jboss-servlet-api_3.0_spec</artifactId>
+                <version>1.0.2.Final</version>
+            </dependency>-->
```

Comment:
What's with this commented out bit?

Reply:
There is no servlet api 5.0 version of that and I don't really understand why z/os would have needed it anyway